src/backend/db/products.js

```diff
@@ -8,23 +8,114 @@ import { v4 as uuid } from "uuid";
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    name: "Aquamarine Skyla Necklace(Sterling Silver)",
+    image: "/assets/images/necklace/necklace1.webp",
+    price: 3535,
+    inStock: true,
+    isBestSeller: false,
+    material: "Aquamarine, Sterling Silver",
+    category: ["Necklace"],
```

Comment:
Suggestion: You can remove this `category` property since you don't have it in other products

Reply:
actually I was adding categories but forgot to add to all. I'll add them later